src/front/app/components/Create/checkboxes.scss

```diff
@@ -0,0 +1,20 @@
+.challenge-create-enabled__title {
+    padding: 20px 0 0 30px;
+    font-weight: bold;
+}
+
+.challenge-create-disabled__title {
```

Comment:
disabled in this case is modificator for title element
will be better to rewrite to
```scss
.challange-create-form {
   &__title {
       padding: 20px 0 0 30px;
       font-weight: bold;
       &_disabled {
          font-family: $checkbox-title-font;
          color: $disabled-point-color;
       }
   }
}
```

and usage 
```js
<div className="challange-create-form__title challange-create-form__title_disabled" >
```